### PR TITLE
desktop: Replacing black and white with opacity changes

### PR DESCRIPTION
**Pull request by pazdera**
Replacing b&w effect with changing opacity and adding the persistent label above them.

Related to: https://github.com/KanoComputing/peldins/issues/2208

cc @tombettany @convolu @alex5imon

### CODE REVIEW

File: kano_init_flow/stages/desktop/main.py [hunk 1]
```diff
@@ -44,7 +44,8 @@ def __init__(self, ctl):
         self._login_launched = False
 
     def first_scene(self):
-        s = self._setup_first_scene()
+        #TODO FIXME PUT BACK s = self._setup_first_scene()
```
Comment (convolu) [inline kano_init_flow/stages/desktop/main.py]: Did you leave this in by mistake?

File: kano_init_flow/stages/desktop/main.py [hunk 2]
```diff
@@ -397,12 +405,20 @@ def _setup_fourth_scene(self):
 
         return scene
 
+    def _desktop_icon_enter_cb(self, widget, event, user=None):
+        if widget.get_opacity() < 1.0:
+            widget.set_opacity(0.75)
+
+    def _desktop_icon_leave_cb(self, widget, event, user=None):
+        if widget.get_opacity() < 1.0:
+            widget.set_opacity(0.5)
```
Comment (convolu) [inline kano_init_flow/stages/desktop/main.py]: Is this supposed to match the value set on line 368? If so would you mind storing it as a "constant" at the top to make it obvious that they are to be the same value?

**Comment by convolu** [comment]
A minor comment towards readability and a question, but other than those it looks ok to me

File: kano_init_flow/stages/desktop/main.py (hunk 1 above)
Comment (pazdera) [inline kano_init_flow/stages/desktop/main.py]: Haha :D Thanks for catching that ... I forgot to remove that comment

**Comment by pazdera** [comment]
@convolu Fixed the problems, thanks!

**Comment by convolu** [comment]
:shipit:

**Pull request merged**